src/features/import-projects/index.ts

```diff
@@ -1,9 +1,18 @@
+import cron from "node-cron";
 import { fetchAndProcessGivethProjects } from "./giveth/helpers";
 
+const task = async () => {
+  console.log("Importing Projects", new Date());
+  await fetchAndProcessGivethProjects();
+};
+
 export const importProjects = async () => {
   try {
-    console.log("Importing Projects");
-    await fetchAndProcessGivethProjects();
+    console.log("Importing Projects has been scheduled.");
+    cron.schedule("0 0 * * *", task, {
```

Comment:
Please read this from config, if it does not exist use the default value. 
Do you think we can find a way to run it by an action?!

Reply:
I totally agree with you, let's have a call and discuss.